src/components/Header.js

```diff
@@ -0,0 +1,23 @@
+import styled from "styled-components";
+function Header() {
+  return (
+    <Head className="header">
+      <h4>Shop App</h4>
```

Comment:
This might semantically wrong. Never skip `header` elements. Read more about [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements?retiredLocale=de)